data/BGC0002757.json

```diff
@@ -0,0 +1,143 @@
+{
+    "changelog": [
+        {
+            "comments": [
+                "Entry submitted",
+                "Entry reviewed and approved.",
+                "Entry reviewed and approved.",
+                "Entry reviewed and approved."
```

Comment:
There's a lot of these duplicated "reviewed" messages, they should be de-duplicated.

Reply:
The reason we have duplicates here is that different people reviewed and approved, and the current schema doesn't really allow to notate this nicely. So we have `comment`s, `contributor`s, and `updated_at` values in three separate lists, with the assumption that they match up.

The only way to fix this would be to have a backwards-incompatible schema change, which I'd like to avoid before changing the schema yet again for MIBiG 4.